actionview/lib/action_view/helpers/form_helper.rb

```diff
@@ -1519,10 +1519,10 @@ def range_field(object_name, method, options = {})
 
       private
         def html_options_for_form_with(url_for_options = nil, model = nil, html: {}, local: !form_with_generates_remote_forms,
-          skip_enforcing_utf8: false, **options)
+          skip_enforcing_utf8: nil, **options)
```

Comment:
No happy about changing this but to leave it as false would mean extra logic in this method to extract the default setting.